src/sheet-cache.ts

```diff
@@ -0,0 +1,113 @@
+/**
+    Copyright 2020 Google LLC
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        https://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+/**
+ * What can be stored in the cache cells
+ */
+export type CacheValue = string|number|null;
```

Comment:
suggestion: split key and value types ( Key and Value types, both extend CacheValue, maybe?)